packages/popover/src/positionUtils.ts

```diff
@@ -139,19 +141,12 @@ const defaultElementPosition = {
 };
 
 /**
- * Returns the boundingWidth if the difference between the boundingWidth and offsetWidth is less than one else the offsetWidth is returned.
+ * Returns the unrounded, floating point width of the element which does not include transformations.
+ *
+ * `offsetWidth` would not work because this returns a rounded number of the element's layout width and `getBoundingClientRect.width` would also not work because it returns an exact number of the rendered width which can include transformations.
  */
-const getClosestToExactWidth = (
-  boundingWidth: number,
-  offsetWidth: number,
-): number => {
-  // offsetWidth returns a rounded number of the element's layout width and height.
-  // boundingWidth returns an exact number with the rendered width and height which can include transformations.
-  // Using the boundingWidth (exact number) is a better indicator of determining if an element will fit in the window, e.g. calcLeft() uses the width to determine the left position and then that number is used to check if it is safetly within the window bounds. In some cases the offsetWidth is rounded up which means that it might not fit in the window bounds when it really can.
-  // With this we check if the difference between the boundingWidth and offsetWidth is less than one, if thats the case then the number was rounded to a whole number and we should use the exact number instead. However if the number is greater than one then that means that the boundingWidth is returning a width that has a transformation applied to it and we don't want that number. We want the untransformed width.
-  const wasRounded = Math.abs(boundingWidth - offsetWidth) < 1;
-
-  return wasRounded ? boundingWidth : offsetWidth;
+const getComputedStyleWidth = (element: HTMLElement): number => {
+  return parseFloat(getComputedStyle(element).width);
 };
 
 export function getElementDocumentPosition(
```

Comment:
An optional comment, but it looks like `getElementDocumentPosition` and `getElementViewportPosition` share a lot of common code. It'd be nice if this were DRY